code/__defines/damage.dm

```diff
@@ -0,0 +1,38 @@
+//gives a baseline for bullet damage
+
+#define	DAM_BULLET_RUBBER 2
+#define	DAM_BULLET_HEAVY_RUBBER 6
+
+#define	DAM_BULLET_PELLET 16
+#define	DAM_BULLET_SHRAP 35
+#define	DAM_BULLET_REVOLVER 50 //Revolvers get snowflake bullets, to keep them relevant
```

Comment:
can't you just use .44 for revolvers? this is a bit confusing

Reply:
![bullet](https://github.com/user-attachments/assets/4ef449e3-4bc7-4c5e-a45d-b328015f868a)
the /obj/item/projectile/bullet/pistol/strong is used by 2 separate calibers 
I didn't want to deal with balancing this weirdness, but I can split it out into separate objects